nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/tools/command/PositionCommand.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Gethin James
+ */
+package org.nuxeo.lib.stream.tools.command;
+
+import java.time.DateTimeException;
+import java.time.Instant;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.nuxeo.lib.stream.computation.Record;
+import org.nuxeo.lib.stream.log.LogLag;
+import org.nuxeo.lib.stream.log.LogManager;
+import org.nuxeo.lib.stream.log.LogOffset;
+import org.nuxeo.lib.stream.log.LogPartition;
+import org.nuxeo.lib.stream.log.LogTailer;
+
+/**
+ * Manipulates the consumer position to the beginning, end or a specific timestamp
+ * @since 10.1
+ */
+public class PositionCommand extends Command {
+
+    protected static final String NAME = "position";
+
+    @Override
+    public String name() {
+        return NAME;
+    }
+
+    @Override
+    public void updateOptions(Options options) {
+        options.addOption(Option.builder("l")
+                .longOpt("log-name")
+                .desc("Log name")
+                .required()
+                .hasArg()
+                .argName("LOG_NAME")
+                .build());
+        options.addOption(
+                Option.builder("g").longOpt("group").desc("Consumer group").hasArg().argName("GROUP").build());
+        options.addOption(Option.builder()
+                .longOpt("reset")
+                .desc("Resets all committed positions for the group")
+                .build());
+        options.addOption(Option.builder()
+                .longOpt("to-end")
+                .desc("Sets the current position to the end for all partitions")
+                .build());
+        options.addOption(Option.builder()
+                .longOpt("to-timestamp")
+                .desc("Sets the current position to a record >= timestamp, eg. " + Instant.now()
```

Comment:
Sets the committed positions for the group to a specific timestamp. The record timestamp used depends ...